.github/workflows/tests.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   run-unit-tests:
 
     runs-on: ubuntu-latest
-    container: compucorp/civicrm-buildkit:1.0.0
+    container: compucorp/civicrm-buildkit:1.1.1-php7.2-chrome
```

Comment:
There is no need for the headless browser for this extension. Can we use the `1.1.1-php7.2` tag instead ?